src/views/settings/Settings.tsx

```diff
@@ -19,6 +21,37 @@ const Settings: FC = () => {
     dispatch,
   ]);
   const handleReset = useCallback(() => dispatch(resetStore()), [dispatch]);
+  const handleExport = async () => {
+    const data = await dataStorage.getItem('persist:data');
```

Comment:
You can use the `useSelector` hook to get the state outside of this function
```suggestion    
  const { data } = useSelector((state) => state);
```

Reply:
Thanks Ricky!